test/etc/tasks/ext/visualizer/VisualizerTaskTest.xml

```diff
@@ -32,15 +32,15 @@
     </target>
 
     <target name="testCustomServer">
-        <!--https://plantuml.telecomsante.com/-->
-        <!--https://plantuml-server.kkeisuke.app/-->
-        <!--http://wiki-plantuml.mercari.in/-->
-        <!--https://plantuml.itemis.de/-->
-        <!--http://gregoryt.fr:8000/plantuml/-->
-        <!--https://swap.sirap.fr/plantuml/-->
-        <!--https://exp-plant.draw.io/plantuml3/-->
-        <!--https://uml.nunction.net/plantuml/-->
-        <visualizer server="https://exp-plant.draw.io/plantuml3/"/>
+        <!-- ℹ️ If test fails, then try using another server: -->
+        <!-- https://plantuml.telecomsante.com/ -->
+        <!-- https://plantuml-server.kkeisuke.app/ -->
+        <!-- http://wiki-plantuml.mercari.in/ -->
+        <!-- https://plantuml.itemis.de/ -->
+        <!-- http://gregoryt.fr:8000/plantuml/ -->
+        <!-- https://swap.sirap.fr/plantuml/ -->
+        <!-- https://exp-plant.draw.io/plantuml3/ -->
+        <visualizer server="https://uml.nunction.net/plantuml/"/>
```

Comment:
we should use a `waitfor` task to ensure the availability of the service instead of the comments. Maybe something like:
```xml
<waitfor>
  <condition property="foo">
    <or>
      <http url="https://uml.nunction.net/plantuml/"/>
      <http url="https://exp-plant.draw.io/plantuml3/"/>
      <!-- ... --> 
    </or>
  </condition>
</waitfor>
```